src/Test.Utilities.Sarif/RandomSarifLogGenerator.cs

```diff
@@ -13,7 +13,7 @@
     /// <summary>
     /// Generate random sarif logs for testing.
     /// </summary>
-    internal static class RandomSarifLogGenerator
+    public static class RandomSarifLogGenerator
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Move this helper to the common Teset.Utilities.Sarif project so it can be used by other test projects.